zebra/zebra_vxlan.c

```diff
@@ -1387,9 +1387,10 @@ static void zvni_print_mac_hash(struct hash_bucket *bucket, void *ctxt)
 			else
 				json_object_int_add(json_mac, "vlan", vid);
 		} else /* No vid? fill out the space */
-			vty_out(vty, " %-5s", "");
-		vty_out(vty, " %u/%u", mac->loc_seq, mac->rem_seq);
+			if (json_mac_hdr == NULL)
```

Comment:
If check for json was missing. We saw seq no being embedded in the json output as well because of a missing "if" check. @srimohans